lib/iris/tests/results/analysis/aggregated_by/multi_missing.cml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube core-dtype="float64" dtype="float64" fill_value="999999.0" long_name="temperature" units="kelvin">
```

Comment:
I think so ... we do want to store the `fill_value`, but only if it's not a default numpy `fill_value`.

There will always be a `fill_value` and this defaults to `None`, which represents the numpy default for the `dtype`. That's assumed, so I don't see the point in peppering the CML with that ... what if numpy changes defaults? That creates carnage in our CML. So in that regard, we're isolated from that ever happening.

So the deal now is that if there is a `fill_value` in the CML, it should be a non-standard numpy `fill_value` that the user cares about, and thus should be stored by exception to the default rule.